src/components/application_manager/test/commands/mobile/get_way_points_request_test.cc

```diff
@@ -77,30 +82,71 @@ class GetWayPointsRequestTest
 
   MockAppPtr mock_app_;
   MessageSharedPtr message_;
-  utils::SharedPtr<commands::GetWayPointsRequest> command_sptr_;
+  utils::SharedPtr<application_manager::commands::GetWayPointsRequest>
+      command_sptr_;
+};
+
+class GetWayPointsRequestTest2
```

Comment:
@OHerasym Please consider more informative name than `GetWayPointsRequestTest2`. Think about the differences with `GetWayPointsRequestTest`.